recipes/flexi-formatter/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "flexi-formatter" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flexi_formatter-{{ version }}.tar.gz
+  sha256: 25e973b73a571d5e3cc23e7da9da616e8484b15c788e83cb1211a5ce8b0cbe16
+
+build:
+  entry_points:
+    - flexi_formatter = flexi_formatter.reformat_flexiplex_tags:app
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin='x.x') }}
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools
+  run:
+    - python >=3.7
+    - typer ==0.12.3
```

Comment:
does this need to be so strictly pinned?

Reply:
You're right, probably not. I'll test it with just python >=3.